Documentation/ResultsStructure.md

```diff
@@ -0,0 +1,150 @@
+
+
+
+## R: results
+
+Results are ordered such that they represent 1 gait cycle, starting at the right side initial contact.
+If *S.misc.gaitmotion_type* is *FullGaitCycle*, the results use *S.solver.N_meshes* meshpoints.
+If *S.misc.gaitmotion_type* is *HalfGaitCycle*, the results use *2\*S.solver.N_meshes meshpoints*.
+
+### R.S: settings used to generate these results
+
+### R.objective
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+|absoluteValues| values of the objective function terms, evaluated at the solution | 1 x 8 | [a.u.] |
+| relativeValues | relative contribution of the objective function terms | 1 x 8 | [%] |
+| labels | names of the objective function terms | 1 x 8 | [text] |
+
+### R.time
+| Field | Description | Dimension | Unit |
+|------ | ----------- | ---- | ---- |
+| mesh | timestamps of mesh points of simulation | 1 x (mesh + 1) | [s] |
+| coll | timestamps of mesh and collocation points of simulation | 1 x (mesh*collocation_order + 1) | [s] |
+| mesh_GC | timestamps of mesh points of gait cycle | 1 x mesh | [s] |
```

Comment:
looking at Dhondt2024_4seg reference results

```suggestion
| mesh_GC | timestamps of mesh points of gait cycle | 1 x (2 *mesh + 1) | [s] |
```

Reply:
because mesh is not S.solver.N_meshes. I put that at the top of the file, because adding a footnote to every single mention of mesh created a huge clutter

Reply:
mesh_GC seems to be 1x201 in Dhondt2024_4seg reference results. Therefore I suggest 
```suggestion
| mesh_GC | timestamps of mesh points of gait cycle | 1 x (2*mesh + 1) | [s] |
```

Reply:
corrected the dimensions of the timestamps